### PR TITLE
implement search call

### LINKED ISSUE #42: Implement real fetch call to get search results
- [x] Verify BE has the functionality to accept selected options and return back only matched applicants.
`GET /api/v1/applicants/search`
- [x] When Search clicked, make sure the fetch only happens if there is at least one option clicked.
- [x] Clean data sent: need to send an array of IDs as the query (for all selected options)
- [x] Write fetch call `getMatchedApplicants` in api-calls (this will replace `getApplicants` that is currently being used) and pass in the cleaned data

### PR DESCRIPTION
**Pull request by GreysonElkins**
#### Type of Change Made 
- [ ] bugfix 
- [x] new feature 
- [ ] breaking change 
- [ ] testing 
- [ ] formatting 
- [ ] styling
#### What's this PR do?
Adds search api call and uses it
#### Where should the reviewer start?
search / searchresults
#### How should this be manually tested?
search for some applicants!
#### Tests are passing
- [x] 
#### Any background context you want to provide?
When you arrive at an applicant page it now fetches the applicant on mount every time. Conditional logic was left in tact for receiving the user as a prop from the search results, but until #62 is resolved we won't be able to use this data-flow 
#### What are the relevant tickets?
close #42

### CODE REVIEW

File: src/assets/api-calls.tsx [hunk 1]
```diff
@@ -12,6 +13,24 @@ export const getApplicants = (): Promise<any> => {
 		})
 }
 
+export const search = 
+	(query: {skills:Array<number>, values: Array<number>}):Promise<any> => {
+		return fetch(`${baseUrl}/applicants/search`, {
+			method: 'POST',
```
Comment (ErinUntermeyer) [inline src/assets/api-calls.tsx:19]: Great call on making this a POST instead of a GET!

**Review by ErinUntermeyer — APPROVED** [review]
Thanks for getting the logic squared away with the search results! Looks great.

File: src/assets/api-calls.tsx (hunk 1 above)
Comment (amykarnaze) [reply src/assets/api-calls.tsx:19]: Agreed!

File: src/assets/user-maker.tsx [hunk 2]
```diff
@@ -4,7 +4,28 @@ const bios = [
   "I'm a wildcard", 
   "I'll rock your world with ruby and rails", 
   "Hire me and I'll make you good money",
-  "If a resume falls in a mailbox and everyone's online to read it, does it make a sound?"
+  "If a resume falls in a mailbox and everyone's online to read it, does it make a sound?",
+  "Designer of Google logo. Fascinated by ancient technology like Blockbuster. Looking for senior full stack position.",
+  "Fascinated by ornithology and avian related legal needs. Author of Bird Law. Philly based. Bird attorney",
```
Comment (amykarnaze) [inline src/assets/user-maker.tsx:9]: Great bios :)

File: src/components/Inbox/Inbox.tsx [hunk 3]
```diff
@@ -35,8 +36,8 @@ const Inbox: React.FC<RouteComponentProps> = (props) => {
           }}>
           <h2>From: {message.employer_name}</h2>
           <h3>{message.employer_email}</h3>
-          <h3>{message.body}</h3>
-          <h5>{message.created_at.substring(0, 11)}</h5>
+          <h5>{moment(message.created_at).format('MMMM DD, YYYY')}</h5>
```
Comment (amykarnaze) [inline src/components/Inbox/Inbox.tsx:39]: Moment rocks!

**Review by amykarnaze — APPROVED** [review]
Way to work in that endpoint and reformat the data. Great idea to use moment to handle that date format for us.

**Pull request merged**